Libraries/Components/SafeAreaView/SafeAreaView.js

```diff
@@ -19,8 +19,6 @@ type Props = $ReadOnly<{|
   emulateUnlessSupported?: boolean,
 |}>;
 
-let exported;
```

Comment:
Since we are following the same pattern in both branches we can still assign to this and then have the one module.exports at the bottom